### ISSUE TITLE
Magic's extraction can't distinguish number of fingers (nf) and multiplier (M) 

### ISSUE THREAD

**Issue by lulu9312** [RTimothyEdwards/magic#207]
I had extracted a spice netlist using magic. The extracted netlist has parameter M but as per the layout it should have been nf. When I tried to simulate the extracted netlist in ngspice it gave incorrect results. 

Attaching the gds file for the block and extracted spice netlist.
[gds and spice file.zip](https://github.com/RTimothyEdwards/magic/files/10301335/gds.and.spice.file.zip)

**Comment by RTimothyEdwards** [comment]
No, magic does not differentiate between number of fingers and M.  Make sure that you understand the difference for BSIM models, however.  W is the _total_ width of all fingers.
